libfreerdp/core/capabilities.c

```diff
@@ -1351,6 +1351,10 @@ static BOOL rdp_apply_input_capability_set(rdpSettings* settings, const rdpSetti
 		}
 		if (settings->HasExtendedMouseEvent)
 			settings->HasExtendedMouseEvent = src->HasExtendedMouseEvent;
+		if (settings->HasRelativeMouseEvent)
+			settings->HasRelativeMouseEvent = src->HasRelativeMouseEvent;
+		if (freerdp_settings_get_bool(settings, FreeRDP_HasQoeEvent))
```

Comment:
Maybe this condition should be extended to also check, whether the fastpath is available (do we know this at this point in the connection sequence already?) (because according to `2.2.7.1.6 Input Capability Set (TS_INPUT_CAPABILITYSET)` there is no slowpath equivalent)?

Reply:
that should be checked if used, not this part here.